webpack/components/extensions/RegistrationCommands/RegistrationCommandsPageHelpers.js

```diff
@@ -0,0 +1,21 @@
+// Activation Keys helpers
+
+export const validateAKField = (hostGroupId, userKeys, hgKeys) => {
+  if (hostGroupId === '') {
+    return (userKeys?.length > 0 ? 'success' : 'error');
+  }
+
+  if (userKeys === undefined && hgKeys === undefined) {
+    return ('default');
+  }
+
+  return ((userKeys?.length > 0 || hgKeys?.length > 0) ? 'success' : 'error');
+};
+
+export const akHasValidValue = (hostGroupId, userKeys, hgKeys) => {
+  if (hostGroupId === '') {
+    return (userKeys?.length > 0);
+  }
+
+  return (hgKeys?.length > 0 || userKeys?.length > 0);
```

Comment:
does it prevent from submitting the registration form?
I am not sure every user will actually need this plugin extension each time,
so maybe it should be `optional`?

Reply:
Yes, it is desired behavior, we don't want to allow submit the form without activation keys.